extensions/stackdriver/stackdriver.cc

```diff
@@ -442,13 +452,35 @@ bool StackdriverRootContext::recordTCP(uint32_t id) {
       record_info.request_info->tcp_connection_state =
           ::Wasm::Common::TCPConnectionState::Open;
       logger_->addTcpLogEntry(*record_info.request_info, peer_node,
-                              record_info.request_info->start_time, outbound);
+                              record_info.request_info->start_time, outbound,
+                              false /* audit */);
       record_info.request_info->tcp_connection_state =
           ::Wasm::Common::TCPConnectionState::Close;
     }
     logger_->addTcpLogEntry(request_info, peer_node,
-                            getCurrentTimeNanoseconds(), outbound);
+                            getCurrentTimeNanoseconds(), outbound,
+                            false /* audit */);
   }
+
+  if (enableAuditLog() && shouldAuditThisRequest()) {
```

Comment:
need to call ::Wasm::Common::populateExtendedRequestInfo(&request_info); here too?